internal/distro/rhel8/pipelines.go

```diff
@@ -1007,6 +1007,17 @@ func ostreeDeployPipeline(
 			},
 		},
 	))
+
+	p.AddStage(osbuild.NewSystemdJournaldStage(
+		&osbuild.SystemdJournaldStageOptions{
+			Filename: "10-persistent.conf",
+			Config: osbuild.SystemdJournaldConfigDropin{
+				Journal: osbuild.SystemdJournaldConfigJournalSection{
+					Storage: osbuild.StoragePresistent,
+				},
+			},
+		},
+	))
```

Comment:
Everything needs to happen _before_ the SELinux stage, or in other words: the SELinux stage needs to be the last one.